src/components/elements/dropdown.tsx

```diff
@@ -97,7 +109,7 @@ const StyledDiv = styled.div`
     top: -30px;
 `
 
-const DropdownSelected = styled.li`
+const DropdownSelected = styled.li<any>`
```

Comment:
![Uploading Screenshot 2022-08-03 at 08.10.59.png…]()

Reply:
@NikitK-deriv which error do you get here?